FordLabs/retroquest
# Theme system settings option

## Overview
Previously, the functionality was added so that when no theme is selected, the theme selected by system preferences will be the default theme (light or dark). This PR adds an option in settings to select system preferences, so if you've already selected light or dark mode, you can change it back to system settings". System settings is still the default if no theme is already set.

Connects #370

### Demo
<img width="494" alt="Screen Shot 2022-06-14 at 4 34 07 PM" src="https://user-images.githubusercontent.com/17144844/173849519-6343a762-b306-40a3-9855-d87fd9d3f64d.png">

## Testing Instructions
1) Clear local storage refresh the page and ensure your system preferences are the default theme for retroquest. (if your computer is set to light mode, the app should be in light mode after you refresh, etc)
2) Go to settings in retroquest and select light or dark mode. Ensure the app changes its settings. 
3) Go to settings in retroquest and select system settings manually. Refresh the page and ensure that the retroquest theme matches your user settings again.
